pkg/controllers/jobset_controller_test.go

```diff
@@ -708,10 +929,23 @@ type makeJobArgs struct {
 	replicas          int
 	jobIdx            int
 	restarts          int
+	parallelism       *int32
+	completions       *int32
+	succeeded         int
+	ready             *int32
```

Comment:
remove those pls, we can set them directly on the wrapperJob returned by makeJob